### DISCUSSION TITLE
Happy to join

### DISCUSSION THREAD

**Discussion by rajchaudar** [community/community#135370] [GitHub Education]
### Hi everyone,

I am very happy to join the GitHub Education community

**Comment by srijon57** [comment]
Hello, my friend. Welcome to the GitHub Education community! It's great to see your enthusiasm!
To join the GitHub Education community, follow these steps:

Apply for the GitHub Student Developer Pack: If you're a student, you can access free tools and resources through the GitHub Student Developer Pack. Just visit (https://education.github.com/pack) and apply with your school-issued email address or student ID.

Explore GitHub Classroom: If you're an educator, GitHub Classroom is a fantastic tool to manage assignments and collaborate with students. You can set up your classroom by visiting (https://classroom.github.com/).

Join Discussions and Forums: Engage with other members of the GitHub Education community by participating in discussions, asking questions, and sharing your knowledge on platforms like (https://github.com/orgs/github-education/discussions).

Stay Updated: Follow (https://twitter.com/GitHubEducation) and keep an eye on the GitHub Education blog for the latest news and opportunities.

If you have any specific questions or need further assistance, feel free to ask. We're here to help you get the most out of your GitHub Education experience!
